projects/packages/forms/src/contact-form/class-util.php

```diff
@@ -386,4 +388,20 @@ public static function get_export_filename( $source = '' ) {
 				sanitize_file_name( $source )
 			);
 	}
+
+	/**
+	 * Get version hash from generated `view.asset.php` files, or fallback to Jetpack version, to be used as cache bust for enqueing JS and CSS files.
+	 *
+	 * @param string $asset_file Path to view.asset.php file.
+	 * @return string either asset hash or Jetpack version.
+	 */
+	public static function get_view_asset_version( $asset_file ) {
+		$asset           = file_exists( $asset_file ) ? require $asset_file : null;
+		$jetpack_version = Constants::get_constant( 'JETPACK__VERSION' );
+		if ( empty( $jetpack_version ) ) {
+			$jetpack_version = '0.1';
+		}
+
+		return ( $asset && ! empty( $asset['version'] ) ) ? $asset['version'] : $jetpack_version;
```

Comment:
Isn't it equivalent to this?

```suggestion
		return ! empty( $asset['version'] ) ? $asset['version'] : $jetpack_version;
```